.github/actions/make/jvm-linux/action.yml

```diff
@@ -0,0 +1,11 @@
+name: Fetch or make the ffi lib for jvm linux
+
+runs:
+  using: composite
+
+  steps:
+      - uses: ./.github/actions/make
+        with:
+          key: jvm-linux-${{ github.sha }}
```

Comment:
So, there are a ton of places where the key depends on `github.sha`. But the reusability there is pretty low.

I wonder how hard it would be to adjust our generic make action to compute a hash based on the inputs of the `make-rule`, and use that for the cache. Possibly that would be better for a separate PR after this. But it's easy to imagine a golden future in which, when you adjust a README or something, all the CI passes in seconds because it just reads the inputs from cached outputs. Or adjusting the wasm-tests reads the cache for ios and android. That kind of thing.